felix/routetable/route_table.go

```diff
@@ -220,6 +221,7 @@ type RouteTable struct {
 
 	// The route deletion grace period.
 	routeCleanupGracePeriod time.Duration
+	featureDetector         environment.FeatureDetectorIface
```

Comment:
CI's failing with:
```
routetable/route_table_test.go:74:4: cannot use dataplane (variable of type *mocknetlink.MockNetlinkDataplane) as environment.FeatureDetectorIface value in argument to NewWithShims: *mocknetlink.MockNetlinkDataplane does not implement environment.FeatureDetectorIface (missing method FeatureGate)
```
I think it should be added to `MockNetlinkDataplane` in https://github.com/projectcalico/calico/blob/master/felix/netlinkshim/mocknetlink/netlink.go